docs/src/gcode/tool-compensation.adoc

```diff
@@ -18,7 +18,7 @@ tool table automatically.
 
 Typical steps for updating the tool table:
 
-* After homing load a tool with 'T__n__ M6' where _n_ is the tool number.
+* After homing load a tool with `T__n__ M6` where _n_ is the tool number.
```

Comment:
That same with double underscores

Reply:
This is perfectly fine with the Visual Studio Code previewer.

What now? When do we plan to transition to asciidoc?

Reply:
Maybe we have to convert all links: https://docs.asciidoctor.org/asciidoc/latest/macros/inter-document-xref/

Reply:
Same here